src/frontend/Page/Docs.elm

```diff
@@ -533,7 +624,7 @@ viewBrowseSourceLinkHelp author project version =
         [ author, project, "tree", V.toString version ]
         []
   in
-  a [ class "pkg-nav-module", href url ] [ text "Browse Source" ]
+  a [ href url ] [ text "Browse Source" ]
```

Comment:
I think it's slightly nicer and more logical to not use a monospace font for "README", "About" and "Source" links as they are not used in source code.

Reply:
Fewer changes per PR is better. Don't want to block the whole thing on deciding this.